test/integration/targets/pull_limit_inventory/runme.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -eux
+
+# http://unix.stackexchange.com/questions/30091/fix-or-alternative-for-mktemp-in-os-x
+MYTMPDIR=$(shell mktemp -d 2>/dev/null || mktemp -d -t 'ansible-testing-XXXXXXXXXX')
+trap 'rm -rf "${MYTMPDIR}"' EXIT
+
+# test for https://github.com/ansible/ansible/issues/13688
+ansible-pull -d "${MYTMPDIR}" -U https://github.com/ansible-test-robinro/pull-integration-test "$@"
```

Comment:
FYI Added to the list of external dependencies on https://public.etherpad-mozilla.org/p/ansible-testing-working-group